### PR TITLE
Peking University is added to institutinos

### PR DESCRIPTION
**Pull request by st3107**
db/institutions.yml is modified: pku is added.

Songsheng Tao

### CODE REVIEW

File: db/institutions.yml [hunk 1]
```diff
@@ -477,4 +477,12 @@ utsinghua:
   state: Haidian
   street: 30 Shuangqing Rd
   zip: 100084
-
+pku:
```
Comment (sbillinge) [inline db/institutions.yml]: please change to pekingu

File: db/institutions.yml [hunk 2]
```diff
@@ -477,4 +477,12 @@ utsinghua:
...
+  aka: [Peking University, PKU]
+  city: Beijing
+  country: China
+  departments: {chemistry: {name: Department of Chemistry}, physics: {name: Department of Physics}}
+  name: Peking University
+  state: ''
+  street: 5 Yiheyuan Road
+  zip: 100871
```
Comment (sbillinge) [inline db/institutions.yml]: please add a carriage return at the end of the line

**Comment by sbillinge** [comment]
@st3107 please can you make the corrections and push again.  Also, you will have to rebase.  Please get help if you are not sure about that.

**Comment by st3107** [comment]
I rebase the branch according to the instruction. And in the terminal, it says "Your branch is up to date with 'origin/master'.". Does it mean success?

**Comment by sbillinge** [comment]
there is still a conflict (you can see it on the GitHub website when you click on the PR.

Did you do ``git fetch upstream master`` before the rebase?

**Comment by sbillinge** [comment]
also, you are trying to rebase onto ``upstream master`` not ``origin master`` which is your fork.

**Comment by st3107** [comment]
I solve the conflicts manually. Though it seems to be fixed, I don't know if this is the right way.

When I try to rebase, there are many merge conflicts (content) probably due to my former mistake operation. I am still trying to figure out how to fix these conflicts.

**Comment by sbillinge** [comment]
This doesn't look quite right so I am reluctant to pull it in.   Please can you work with @chiahaoliu to fix this up?  The easiest may be to blow away this PR and start over, but work with Chia Hao to figure it out.

**Comment by chiahaoliu** [comment]
@st3107 yeah, merging will solve the conflict but it will add another commit and further divert you from the upstream working tree. Could you drop by office some time? I think I can show you pretty quick.

**Comment by sbillinge** [comment]
songsheng is stuck in China, so won't be in the office until later this month.  We can go over the rebasing with him at that point.  No rush.

S

**Pull request closed without merging**